.travis.yml

```diff
@@ -35,6 +35,9 @@ matrix:
         - MATRIX_EVAL="CC=clang-5.0 && CXX=clang++-5.0"
 
 before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then wget -q ${SDE_URL}                            ; fi
```

Comment:
Where does SDE_URL come from? Is it a Travis env variable?

Reply:
SDE_URL is a Travis secure environment variable. Since redistribution of SDE is not allowed, we need to hide where we are downloading the SDE package from.

https://github.com/shibatch/sleef/issues/85